modules/network-observability-total-resource-usage.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+// * network_observability/configuring_operator.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="network-observability-total-resource-usage-table_{context}"]
+= Total average memory and CPU usage
+The following table outlines averages of total resource usage for clusters with a sampling sizes of 1, 50, and 400.
```

Comment:
Not sure if "sampling size" is correct , I would tend to say "sampling value" and not consider it as a _size_, wdyt? Same for the table header where it's "Sample size".

An alternative would be "Sampling ratio" and present it as 1:1, 1:50, 1:400

Reply:
I like sampling value instead of sampling size here. Thanks for the options for different wording :)

Reply:
Thank you! Updated the headers and the paragraph with the "Sampling value" terminology.